projects/step-core/src/lib/components/autorefresh-toggle/autorefresh-toggle.component.html

```diff
@@ -1,4 +1,4 @@
-<button mat-icon-button [matMenuTriggerFor]="menu" [matTooltip]="selectedInterval.label">
+<button mat-icon-button [matMenuTriggerFor]="menu" [matTooltip]="selectedInterval.label" class="grey-border z-1000">
```

Comment:
Isn't the same could be achieved with `mat-raised-button`?

Reply:
Not really. This is result of adding mat-raised-button: 
![image](https://github.com/exense/step-frontend/assets/158443736/03b5986f-0973-45fc-a6f4-3eee22db649c)